src/site/content/en/blog/lighthouse-ci/index.md

```diff
@@ -0,0 +1,351 @@
+  - performance
+---
+
+[Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci) is a suite of tools for using Lighthouse during continuous integration. Lighthouse CI can be incorporated into developer workflows in many different ways. This guide covers the following topics:
+
+*   Using the Lighthouse CI CLI.
+*   Configuring your CI provider to run Lighthouse CI.
+*   Setting up a [GitHub Action](https://github.com/features/actions) and [status check](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) for Lighthouse CI. This will automatically display Lighthouse results on GitHub pull requests.
+*   Building a performance dashboard and data store for Lighthouse reports.
+
+
+## Overview
+
+Lighthouse CI is a suite of free tools that facilitate using Lighthouse for performance monitoring. A single Lighthouse report provides a snapshot of a web page's performance at the time that it is run; Lighthouse CI shows how these findings have changed over time. This can be used to identify the impact of particular code changes or ensure that performance thresholds are met during continuous integration processes. Although performance monitoring is the most common use case for Lighthouse CI, it can be used to monitor other aspects of the Lighthouse report - for example, SEO or accessibility.
+
+The core functionality of Lighthouse CI is provided by the Lighthouse CI command line interface. (Note: This is a separate tool than the [Lighthouse CLI](https://github.com/GoogleChrome/lighthouse#using-the-node-cli).) The Lighthouse CI CLI provides a set of [commands](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#commands) for using Lighthouse CI. For example, the `autorun` command executes multiple Lighthouse runs, identifies the median Lighthouse report, and uploads the report for storage. This behavior can be heavily customized by passing additional flags or customizing Lighthouse CI's configuration file, `lighthouserc.js`.
+
+Although the core functionality of Lighthouse CI is primarily encapsulated in the Lighthouse CI CLI, Lighthouse CI is typically used through one of the following approaches:
+
+*   Running Lighthouse CI as part of continuous integration
+*   Using a Lighthouse CI GitHub action that runs and comments on every pull request
+*   Tracking performance over time via the dashboard provided by Lighthouse Server.
+
+All of these approaches are built upon the Lighthouse CI CLI.
+
+Alternatives to Lighthouse CI include third-party performance monitoring services or writing your own script to collect performance data during the CI process. You should consider using a third-party service if you'd prefer to let someone else handle the management of your performance monitoring server and test devices, or, if you want notification capabilities (such as email or Slack integration) without having to build these features yourself.
+
+## Use Lighthouse CI locally
```

Comment:
```suggestion
## Use Lighthouse CI locally {: #cli }
```

I like to add custom IDs so that if anyone deep-links to the section they get a nice, concise link that doesn't change over time